databases/mongo.md

```diff
@@ -26,17 +26,14 @@ require 'sinatra'
 require 'mongo'
 require 'json/ext' # required for .to_json
 
-include Mongo
-
 configure do
-  conn = MongoClient.new("localhost", 27017)
-  set :mongo_connection, conn
-  set :mongo_db, conn.db('test')
+  db = Mongo::Client.new([ '127.0.0.1:27017' ], :database => 'test')  
+  set :mongo_db, db
```

Comment:
I feel it would be worth changing this to `set :mongo_db, db['test']` and get rid of all the `mongo_db['test']` calls.